src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -108,70 +111,105 @@ export function DiscoverField({
     }
   };
 
+  function togglePopover() {
+    setOpen(!infoIsOpen);
+  }
+
   function wrapOnDot(str?: string) {
     // u200B is a non-width white-space character, which allows
     // the browser to efficiently word-wrap right after the dot
     // without us having to draw a lot of extra DOM elements, etc
     return str ? str.replace(/\./g, '.\u200B') : '';
   }
 
+  const dscFieldIcon = (
+    <FieldIcon type={field.type} label={getFieldTypeName(field.type)} scripted={field.scripted} />
+  );
+
+  const fieldName = (
+    <span
+      data-test-subj={`field-${field.name}`}
+      title={field.name}
+      className="dscSidebarField__name"
+    >
+      {useShortDots ? wrapOnDot(shortenDottedString(field.name)) : wrapOnDot(field.displayName)}
+    </span>
+  );
+
+  let actionButton;
+  if (field.name !== '_source' && !selected) {
```

Comment:
there's one regression concerning the action button. when you navigate the sidebar fields with tab, then a field has focus, the action button is no longer displayed:
<img width="232" alt="Bildschirmfoto 2020-08-05 um 18 12 52" src="https://user-images.githubusercontent.com/463851/89441557-a2ae3180-d74d-11ea-9abc-3650389ac785.png">

Reply:
Good catch, I'll look into it

Reply:
<img width="281" alt="image 12" src="https://user-images.githubusercontent.com/4016496/89465481-083af780-d727-11ea-8ea8-46f2200c9cbf.png">

@kertal this should be fixed now

Reply:
yes, thx, works 👍